### ISSUE TITLE
Plugin crash when hub is offline

### ISSUE THREAD

**Issue by mbriney** [aarons22/homebridge-bond#64]
**Describe the bug**
The plugin crashes when the hub goes offline.  It then causes Homebridge to not load.

```
Feb  1 15:46:50 homebridge homebridge[1372]: at BondPlatform.bondForDevice (/usr/lib/node_modules/homebridge-bond/dist/index.js:189:23)
Feb  1 15:46:50 homebridge homebridge[1372]: at BondPlatform.setupObservers (/usr/lib/node_modules/homebridge-bond/dist/index.js:139:27)
Feb  1 15:46:50 homebridge homebridge[1372]: at Timeout.setInterval [as _onTimeout] (/usr/lib/node_modules/homebridge-bond/dist/index.js:131:26)
Feb  1 15:46:50 homebridge homebridge[1372]: at ontimeout (timers.js:498:11)
Feb  1 15:46:50 homebridge homebridge[1372]: at tryOnTimeout (timers.js:323:5)
Feb  1 15:46:50 homebridge homebridge[1372]: at Timer.listOnTimeout (timers.js:290:5)
Feb  1 15:46:50 homebridge homebridge[1372]: [2020-2-1 15:46:50] [Bond] Configure Accessory: Guest Bedroom CF
Feb  1 15:46:50 homebridge homebridge[1372]: [2020-2-1 15:46:50] Error: No Bond found for device Ceiling fan
Feb  1 15:46:50 homebridge homebridge[1372]: at BondPlatform.bondForDevice (/usr/lib/node_modules/homebridge-bond/dist/index.js:189:23)
Feb  1 15:46:50 homebridge homebridge[1372]: at BondPlatform.setupObservers (/usr/lib/node_modules/homebridge-bond/dist/index.js:139:27)
Feb  1 15:46:50 homebridge homebridge[1372]: at Timeout.setInterval [as _onTimeout] (/usr/lib/node_modules/homebridge-bond/dist/index.js:131:26)
Feb  1 15:46:50 homebridge homebridge[1372]: at ontimeout (timers.js:498:11)
Feb  1 15:46:50 homebridge homebridge[1372]: at tryOnTimeout (timers.js:323:5)
Feb  1 15:46:50 homebridge homebridge[1372]: at Timer.listOnTimeout (timers.js:290:5)
Feb  1 15:46:50 homebridge homebridge[1372]: at listOnTimeoutNT (timers.js:369:26)
Feb  1 15:46:50 homebridge homebridge[1372]: at _combinedTickCallback (internal/process/next_tick.js:136:11)
Feb  1 15:46:50 homebridge homebridge[1372]: at process._tickCallback (internal/process/next_tick.js:181:9)
```

**Expected behavior**
Ideally, if the hub wasn't found it would just persist in homebridge until the connection could be re-established.

**Information (please complete the following information):**
 - Bond firmware version: 2.10.8

**Comment by vdotn** [comment]
I just experienced this today and came here to suggest the same. +1 from me!

**Comment by dxdc** [comment]
This error is related (in a way) to #66

Same error, but different reason that it appears.

**Comment by LeviJSmith** [comment]
I have this problem as well.

**Comment by LeviJSmith** [comment]
Any news on a fix fir this.

**Comment by aarons22** [comment]
Hey everyone! Finally got around to addressing this issue. This should be fixed in [2.5.6](https://github.com/aarons22/homebridge-bond/releases/tag/2.5.6)
